scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* eslint-env node */
+
+require('./util/work-from-root')();
+
+var docBuilder = require('./util/doc-builder')({
```

Comment:
What version of node is your project set to use locally? Seems like you should be able to use things like `const` and `class` throughout these scripts.